docs/_posts/gadde5300/2021-12-10-bert_hi_en_ner_hi.md

```diff
@@ -0,0 +1,130 @@
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/bert_hi_en_ner_hi_3.2.0_3.0_1639158216255.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler()\
+    .setInputCol('text')\
+    .setOutputCol('document')
+
+sentence_detector = SentenceDetector() \
+    .setInputCols(['document'])\
+    .setOutputCol('sentence')
+
+tokenizer = Tokenizer()\
+    .setInputCols(['sentence']) \
+    .setOutputCol('token')
+
+tokenClassifier_loaded = BertForTokenClassification.pretrained("bert_hi_en_ner")\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+        .setInputCols(["sentence","token","ner"])\
+        .setOutputCol("ner_chunk")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, tokenClassifier_loaded, ner_converter])
+
+pipeline_model = nlp_pipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+
+result = pipeline_model.transform(spark.createDataFrame([['रिलायंस इंडस्ट्रीज़ लिमिटेड एक भारतीय संगुटिका नियंत्रक कंपनी है, जिसका मुख्यालय मुंबई, महाराष्ट्र में स्थित है।रतन नवल टाटा (28 दिसंबर 1937, को मुम्बई, में जन्मे) टाटा समुह के वर्तमान अध्यक्ष, जो भारत की सबसे बड़ी व्यापारिक समूह है, जिसकी स्थापना जमशेदजी टाटा ने की और उनके परिवार की पीढियों ने इसका विस्तार किया और इसे दृढ़ बनाया।']], ["text"]))
+
+```
+```scala
+val document_assembler = DocumentAssembler() 
+    .setInputCol("text") 
+    .setOutputCol("document")
+
+val sentence_detector  = SentenceDetector()
+    .setInputCols("document")
+    .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()\
+    .setInputCols("sentence") 
+    .setOutputCol("token")
+
+val tokenClassifier_loaded  = BertForTokenClassification.pretrained(""bert_hi_en_ner"")
+  .setInputCols("sentence","token")
+  .setOutputCol("ner")
+
+val ner_converter = NerConverter()
+   .setInputCols("sentence","token","ner")
+    .setOutputCol("ner_chunk")
+
+val nlp_pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, tokenClassifier_loaded, ner_converter))
+
+val data = Seq("'रिलायंस इंडस्ट्रीज़ लिमिटेड एक भारतीय संगुटिका नियंत्रक कंपनी है, जिसका मुख्यालय मुंबई, महाराष्ट्र में स्थित है।रतन नवल टाटा (28 दिसंबर 1937, को मुम्बई, में जन्मे) टाटा समुह के वर्तमान अध्यक्ष, जो भारत की सबसे बड़ी व्यापारिक समूह है, जिसकी स्थापना जमशेदजी टाटा ने की और उनके परिवार की पीढियों ने इसका विस्तार किया और इसे दृढ़ बनाया।'").toDF("text")
+
+val result = pipeline.fit(data).transform(data)
+
+```
+</div>
+
+## Results
+
+```bash
++---------------------------+--------------------------------+
```

Comment:
Can you format this a little bit better? Also, is this showing English capabilities?